src/Transitions/Transition.js

```diff
@@ -0,0 +1,173 @@
+'use strict';
+import React from 'react';
+import TransitionEvents from '../utils/TransitionEvents';
+import classnames from 'classnames';
+
+function omit(obj, keys) {
+  let included = Object.keys(obj).filter( k => keys.indexOf(k) === -1)
+  let newObj = {};
+
+  included.forEach( key => newObj[key] = obj[key] )
+  return newObj;
+}
```

Comment:
this is good candidate to use `lodash` :wink: 
`lodash` performance is faster
![screen shot 2015-06-26 at 5 21 05 pm](https://cloud.githubusercontent.com/assets/847572/8379332/f155e88c-1c27-11e5-8795-51a55d41f814.png)